spec/features/number_fields_spec.rb

```diff
@@ -4,9 +4,9 @@
 INVALID_INTEGER_VALUES = ["two", "2 00", "2.37"].freeze
 INVALID_MONEY_VALUES = ["two", "2 00", "$54"].freeze
 VALID_MONEY_VALUES = [
-  ["2,000", "2,000"],
+  ["2,000", "2,000", "2,000 "],
```

Comment:
If you look at how VALID_MONEY_VALUES gets used in the tests, only the first and second array elements of each sub-array ever get read. The elements you've added will be ignored.

What you want to do is add a new sub-array with the first (what gets written) having a space, and the second (what is expected to be read) not having one.

And since we've discovered that the formatting here is confusing, maybe worth changing these sub-arrays to hashes with `input` and `output` as the keys?